config/redirects.txt

```diff
@@ -13,3 +13,6 @@
 ^\/getting-started -> /raspberrypi/nodejs/getting-started
 
 ^\/configuration\/resin-base-images(\/|$) -> /runtime/resin-base-images
+
+# Renamed pages
+^\/deployment\/wifi\/?(.*)? -> /deployment/network/$1
```

Comment:
it works and seems to be equivalent to this: `^\/deployment\/wifi(\/?(#.*)?)$`

because:
* `a|ab` is equivalent to `ab?`
* `[\/]` is equivalent to `\/` (because you don't actually need the backward slash in the character class, and you don't need the class for a single character)

I would probably also replace `(#.*)` with `(#.+)` because we only care about non-empty fragments